CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ target_link_libraries( ${PROJECT_NAME} PRIVATE ${JULIA_LIBRARY} MPI::MPI_C )
 
 # Set appropriate compile flags
 target_compile_options( ${PROJECT_NAME} PUBLIC "-fPIC" )
+target_compile_options( ${PROJECT_NAME} PRIVATE -Wall -Wextra -Werror)
```

Comment:
This will add the flags regardless of release or debug build type, right? 
But this is reasonable anyway.

Reply:
Correct. But why wouldn't we 😬?